sources/ClangSharp/Generated/CXIdxObjCInterfaceDeclInfo.cs

```diff
@@ -1,11 +1,14 @@
-using System;
-
 namespace ClangSharp
 {
-    public partial struct CXIdxObjCInterfaceDeclInfo
+    public unsafe partial struct CXIdxObjCInterfaceDeclInfo
     {
-        public IntPtr containerInfo;
-        public IntPtr superInfo;
-        public IntPtr protocols;
+        [NativeTypeName("const CXIdxObjCContainerDeclInfo *")]
+        public CXIdxObjCContainerDeclInfo* containerInfo;
+
+        [NativeTypeName("const CXIdxBaseClassInfo *")]
+        public CXIdxBaseClassInfo* superInfo;
```

Comment:
It also allows you to see the "const" info, which can be very useful for exposing the right managed API (and knowing things like: "should this be span or rospan").